crates/store/re_types_core/src/id.rs

```diff
@@ -111,8 +115,6 @@ impl std::ops::DerefMut for ChunkId {
     }
 }
 
-crate::delegate_arrow_tuid!(ChunkId as "rerun.controls.ChunkId");
```

Comment:
We still need this in some form or another -- ChunkIds as a column are used all over the place in the DP's  indexes